web/src/lib/api-client/apiClient.ts

```diff
@@ -46,11 +46,12 @@ export const checkElevatorViolations = (address: string, municipalityId: string)
   return post(`/elevator-safety/violations`, { address, municipalityId });
 };
 
-export const checkHousingHotelMotelRegistrationStatus = (
+export const checkHousingRegistrationStatus = (
   address: string,
-  municipalityId: string
+  municipalityId: string,
+  propertyInterestType: string
```

Comment:
[dust] Should this be a specific set of strings we anticipate?

Reply:
Yeah, makes sense, I'll update this